test/smoke_spec.js

```diff
@@ -0,0 +1,73 @@
+var PageObject = require('../page_objects/page_object_model')
+
+describe('smoke test demo', function() {
+var pageObject
+
+beforeEach(function(){
+
+  pageObject = new PageObject()
+
+  browser.waitForAngularEnabled(false);
+
+  browser.get('http://automationpractice.com/index.php');
```

Comment:
Can you make one separated folder named "Data" with one data.js class where you will store every data you use (every link, input, text etc).
Reason: If at some point data changes, you do not need to change your test, only data class.